doc/shapes/2025.02/add_support_for_library_publication.adoc

```diff
@@ -0,0 +1,72 @@
+= Add support for library publication
+
+== Problem
+
+SysON doesn't provide an  **in-application** way to import models from other projects.
+The current import/export mechanisms rely on generating a textual SysML file, and uploading such file in a project to import it.
+
+When exported models are used as shared reusable components, it is the responsibility of the stakeholders to:
+- Store the shared models in an external location
+- Ensure that every user is working on the same version of the shared model
+- Coordinate to export a new version when needed, and import it in all the dependent projects
+
+These processes are error-prone, and could be more controlled if they all happened inside the application.
+
+== Key Result
+
+SysON users should be able to:
+- Publish a model with a given version number
+
+== Solution
+
+The contextual menu of models containing _LibraryPackage_ elements in the explorer contains a new _Publish_ tool.
+This tool opens a pop-up that asks the user for a version number and a description.
```

Comment:
I chose to put the _Publish_ tool in the contextual menu of the **model** (not the LibraryPackage). This implies that 

- a model containing multiple LibraryPackage will be published as a single artifact (it won't be possible to import just 1 library of this model, users will have to import all of them)
- a model containing LibraryPackage and regular Package cannot be published (the model has to contain only LibraryPackage element)

This is aligned with how we manage user libraries in SysON, but we can discuss it if it is too restrictive.